File: answers/C/@mykeels/01-name-concatenation.c
#include <stdio.h>
#include <string.h>

int main() {
    char firstName[50], lastName[50];
    int age = 0;
    
    printf("Enter your first name: ");
    scanf("%s", firstName);
    
    printf("Enter your last name: ");
    scanf("%s", lastName);

    printf("Enter your age: ");
    scanf("%d", &age);

    printf("\nWelcome, %s %s (%d)\n", firstName, lastName, age);
    
    return 0;
}

File: answers/Go/@codehakase/01-name-concatenation.go
package main

import "fmt"

func main() {
	var (
		firstname string
		lastname  string
		age       int
	)
	fmt.Print("Enter first name: ")
	fmt.Scanf("%s\n", &firstname)

	fmt.Print("Enter last name: ")
	fmt.Scanf("%s\n", &lastname)

	fmt.Print("Enter your age: ")
	fmt.Scanf("%d\n", &age)
	fmt.Print("\n")

	fmt.Printf("Welcome, %s %s (%d)\n", firstname, lastname, age)
}
